docs/features/notifications.md

```diff
@@ -50,6 +50,22 @@ stringData:
 
 Learn more about supported services and configuration settings in services [documentation](../generated/notification-services/overview.md).
 
+## Namespace based configuration
+
+!!! important
+Available since v1.6
+
+A common installation method for Argo Rollouts is to install it in a dedicated namespace to manage a whole cluster. In this case, the administrator is the only
+person who can configure notifications in that namespace generally. However, in some cases, it is required to allow end-users to configure notifications
+for their Rollout resources. For example, the end-user can configure notifications for their Rollouts in the namespace where they have access to and their rollout is running in.
+
+To use this feature all you need to do is create the same configmap named `argo-rollouts-notification-configmap` and possibly 
```

Comment:
Maybe add a link to where this configmap is defined so users have a easy reference to how this configmap is supposed to look like

Reply:
In context of the rest of the doc I think the idea of the config map is explained: https://argo-rollouts.readthedocs.io/en/stable/features/notifications/

Reply:
The documentation mentions about the `argo-rollouts-notification-configmap` and shows some simple examples. I  think that we should provide a link to the configmap definition with all possible fields it can have. Looking at the doc , I couldn't find such info easily. Is there a place users can look to see the full reference doc?